src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -3078,8 +3078,8 @@ theorem getMsbD_rotateLeft_of_lt {n w : Nat} {x : BitVec w} (hi : r < w):
       by_cases h₁ : n < w + 1
       · simp only [h₁, decide_true, Bool.true_and]
         have h₂ : (r + n) < 2 * (w + 1) := by omega
+        rw [← Nat.sub_mul_eq_mod_of_lt_of_le (i := (r + n)) (w := (w + 1)) (n := 1) (by omega) (by omega)]
         congr 1
-        rw [← Nat.sub_mul_eq_mod_of_lt_of_le (n := 1) (by omega) (by omega), Nat.mul_one]
```

Comment:
Is this change needed?

Reply:
There is still a change in this line. Do we need this change at all? It does not really seem to make anything simpler and adds noise that can be avoided, no?

Reply:
I now simplified it further and removed the unnecessary specification. theorem `Nat.mul_one` is not necessary here, but if this change is too much happy to revert it.

Reply:
I think the current version is fine